android/assets/jsons/translations/Dutch.properties

```diff
@@ -1365,13 +1293,11 @@ wounded units = gewonde eenheden
 Wounded = Gewond
 
 # For the "All newly-trained [relevant] units in this city receive the [] promotion" translation. Relevant as in 'units that can receive'
-relevant = relevante
+relevant = relevantie
```

Comment:
Ik zou het hier wel als 'relevante' laten staan, in het commentaar erboven zie je de zin waarin het gebruikt wordt, en daar is het echt wel een bijvoegelijk naamwoord.